fjord/feedback/views.py

```diff
@@ -98,7 +98,7 @@ def thanks_view(request):
             pass
 
     if feedback:
-        product_slug = feedback.product
```

Comment:
All the remediations for slug/name should be backed out until code is rebased onto code from bug 1236581.